internal/rbd/rbd_attach.go

```diff
@@ -409,6 +409,21 @@ func createPath(ctx context.Context, volOpt *rbdVolume, device string, cr *util.
 	}
 	devicePath := strings.TrimSuffix(stdout, "\n")
 
+	if isNbd && volOpt.isEncrypted() {
+		// FIXME: Just a test
```

Comment:
Instead of `dd`, you should prefer `wipefs`. And well, the e2e RBD images are newly allocated, so they should not contain any data... But I guess you can see that from the results of the CI jobs too now.

Reply:
Sure, thanks!